test/test_classifier.rb

```diff
@@ -47,6 +47,9 @@ def test_classify_ambiguous_languages
     Samples.each do |sample|
       language  = Linguist::Language.find_by_name(sample[:language])
       languages = Language.find_by_filename(sample[:path]).map(&:name)
+      next unless languages.length != 1
```

Comment:
Double negatives are confusing. Can you switch this to `next if languages.length == 1`